### PR TITLE
[BUG] - Fix Calendar Element Models

### PR DESCRIPTION
**Pull request by guaj**
- Made `eventId` optional
- Find events based on Mongo assigned `_id` attribute rather than `eventId`
- Removed event return when deleting event (not being used)

### CODE REVIEW

**Comment by guaj** [comment]
Bugfixes for the front end calendar events modification features to work.

File: src/models/event.model.js [hunk 1]
```diff
@@ -18,7 +18,7 @@ const ReccurencyTypes = {
 
 const eventSchema = new Schema({
     username: { type: String, required: true },
-    eventID: { type: String, required: true, unique: true, sparse: true },
+    eventID: { type: String, unique: true, sparse: true },
```
Comment (mnashat) [inline src/models/event.model.js:21]: Why is this field still here? Aren't we using _id for events now?

**Review by mnashat — CHANGES_REQUESTED** [review]

File: src/models/event.model.js (hunk 1 above)
Comment (guaj) [reply src/models/event.model.js:21]: we can open a new issue to remove this field.

Comment (mnashat) [reply src/models/event.model.js:21]: Noted, approved.

**Review by mnashat — APPROVED** [review]

**Review by kikkiii21 — APPROVED** [review]

**Pull request merged**